.github/workflows/bump-version.yml

```diff
@@ -0,0 +1,52 @@
+name: Bump Version on Release Branch Creation
+
+on:
+  create:
+    branches:
+      - 'release/*'
+
+jobs:
+  bump-version:
+    runs-on: ubuntu-latest
+    if: startsWith(github.ref, 'refs/heads/release/')
+    
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
```

Comment:
Shouldn't we specify ref: master here? We want to make a change on master branch but triggering condition here will probably checkout current ref not default ref (so in this case release branch not master).